src/Components/ArticleCard/ArticleCard.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import {
+  Card,
+  Title,
+  Author,
+  CreatedAt,
+  Image,
+  Topic,
+  Votes,
+  CommentCount,
+} from "./ArticleCard.styled";
+
+function ArticleCard({
+  article_id,
+  title,
+  author,
+  created_at,
+  votes,
+  article_img_url,
+  topic,
+  comment_count,
+}) {
+  return (
+    <Link to={`/articles/${article_id}`}>
```

Comment:
lets not wrap everything inside the link to the article because => when you click on the user you might want to see the users articles. when you click on the topic you might want to see the articles under that topic.